doc/nrf/app_dev/device_guides/nrf54l/kmu_provision.rst

```diff
@@ -73,6 +73,13 @@ Once you have an unprovisioned SoC, upload keys to the board by running the foll
 
 * Parameter ``--dev-id`` specifies the interface serial number and should be used if multiple J-link interfaces are connected to the development machine.
 
+* Parameter ``-p" (--policy)`` specifies the policy applied to the given set of keys. 
+  You can apply the following options:
+   * ``lock-last`` - Uploads the last key as locked,  while the preceding keys are revocable. 
+     This option is set by default.
+   * ``revocable`` - Enables revocation for each key.
```

Comment:
Script supports `revokable` as CLI parameter name. This needs to be aligned too.